### ISSUE TITLE
Charts broken in version 2.2.31

### ISSUE THREAD

**Issue by jsofra** [clj-pdf/clj-pdf#156]
As of version 2.2.31 the charting examples in the readme no longer work.

```clojure
> (require '[clj-pdf.core :as clj-pdf])
> (clj-pdf/pdf 
    [{}
      [:chart {:type :line-chart :title "Line Chart" :x-label "checkpoints" :y-label "units"}
        ["Foo" [1 10] [2 13] [3 120] [4 455] [5 300] [6 600]]
        ["Bar" [1 13] [2 33] [3 320] [4 155] [5 200] [6 300]]]]
    "doc.pdf")

{:Stylesheet nil, :references nil, :left-margin 36.0, :right-margin 36.0, :top-margin 36.0, :bottom-margin 56.0, :page-width 595.0, :page-height 842.0, :pdf-writer #object[cljpdf.text.pdf.PdfWriter 0x2d7545e7 "cljpdf.text.pdf.PdfWriter@2d7545e7"]} [:chart {:type :line-chart, :title "Line Chart", :x-label "checkpoints", :y-label "units"} ["Foo" [1 10] [2 13] [3 120] [4 455] [5 300] [6 600]] ["Bar" [1 13] [2 33] [3 320] [4 155] [5 200] [6 300]]]
Exception Unsupported chart type   clj-pdf.charting/chart (charting.clj:117)
```

**Comment by yogthos** [comment]
Looks like I missed a bug in [this refactor](https://github.com/clj-pdf/clj-pdf/commit/4e391d96919547f3f68cc11a3a4309bba0566d1b#diff-b81b76f00afe0c03ac9ee79e2f4cb9f6R117). I just pushed out a fix with `2.2.32`, let me know if everything looks good on your end.

**Comment by yogthos** [comment]
Ok that should be fixed as well. A type annotation got added for `XYLineAndShapeRenderer`, but looks like that renderer doesn't support all the options. Pushed out `2.2.33` with the fix.
